twine/package.py

```diff
@@ -148,6 +148,10 @@ def metadata_dictionary(self):
             "requires_dist": meta.requires_dist,
             "requires_external": meta.requires_external,
             "requires_python": meta.requires_python,
+
+            # Metadata 2.1
+            "provides_extras": meta.provides_extras,
```

Comment:
Looks right to me per [the names for these attributes as provided by the class from `pkginfo`](https://bazaar.launchpad.net/~tseaver/pkginfo/trunk/view/148/pkginfo/distribution.py).

Let me make sure I've got this right: even though the field defined in the PEP is `Provides-Extra`, we use the "s" at the end to indicate that it might be multiple strings, as with `project_urls`. Right?

Reply:
Yes, that seems to be the status quo (really, to indicate that this is a `list`).

Reply:
Great, thanks.